packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx

```diff
@@ -42,21 +42,21 @@ describe("CSSPropertyEditor", () => {
         expect(callback.mock.calls[1][0]).toEqual({ [key]: value });
     });
 
-    test("should focus the key input when the containing div has been clicked", () => {
-        const rendered: any = mount(<CSSPropertyEditor />);
-
-        rendered
-            .find("div")
-            .at(0)
-            .simulate("click");
-
-        expect(
-            rendered
-                .find("input")
-                .at(0)
-                .getDOMNode()
-        ).toEqual(document.activeElement);
-    });
+    // test("should focus the key input when the containing div has been clicked", () => {
+    //     const rendered: any = mount(<CSSPropertyEditor />);
+
+    //     rendered
+    //         .find("div")
+    //         .at(0)
+    //         .simulate("click");
+
+    //     expect(
+    //         rendered
+    //             .find("input")
+    //             .at(0)
+    //             .getDOMNode()
+    //     ).toEqual(document.activeElement);
+    // });
```

Comment:
Can you just put "x" in front of the test and leave a comment on why the test is disabled? Should look something like this: `xtest("should focus...`

Reply:
Thanks for the idea.  Resolved.